src/figma/tokens.json

```
@@ -329,232 +329,232 @@
         "muted": {
           "value": "#BBC0C5",
-          "description": "(grey200: #BBC0C5) icon.muted should be used for all low priority icon used on background.default or background.alternative. These could also include placeholder or inactive icons. It will also not meet AAA or AA accessibility standards.",
+          "description": "(grey200: #BBC0C5) For inactive or low priority icons.",
           "type": "color"
         }
       },
       "border": {
         "default": {
           "value": "#BBC0C5",
-          "description": "(grey200: #BBC0C5) border.default should be used for all neutral component border colors such as cards or inputs in default state.",
+          "description": "(grey200: #BBC0C5) For default neutral borders with visible contrast. (Example: text inputs)",
           "type": "color"
         },
         "muted": {
           "value": "#D6D9DC",
-          "description": "(grey100: #D6D9DC) border.muted should be used for any borders with low contrast or subtle definition like dividers",
+          "description": "(grey100: #D6D9DC) Weaker contrast option for neutral borders. (Example: dividers)",
           "type": "color"
         }
       },
       "overlay": {
         "default": {
           "value": "#00000099",
-          "description": "(black000: #000000 60% opacity) overlay.default should be used for all general overlay backgrounds",
+          "description": "(black000: #000000 60% opacity) For shading layers behind modality screens.",
           "type": "color"
         },
         "alternative": {
           "value": "#000000CC",
-          "description": "(black000: #000000 80% opacity) overlay.alternative should be used for all general overlay backgrounds that need a higher contrast than overlay.default",
+          "description": "(black000: #000000 80% opacity) Added contrast option of overlay.",
           "type": "color"
         },
         "inverse": {
           "value": "#FCFCFC",
-          "description": "(white010: #FCFCFC) overlay.inverse should be used only as the foreground element on top of overlay.default used for text or icons",
+          "description": "(white010: #FCFCFC) [Deprecated] overlay.inverse should be used only as the foreground element on top of overlay.default used for text or icons",
           "type": "color"
         }
       },
       "primary": {
         "default": {
           "value": "#037DD6",
-          "description": "(blue500: #037DD6) primary.default should be used for all primary action components such as buttons or links.",
+          "description": "(blue500: #037DD6) For interactive or active semantic",
           "type": "color"
         },
         "alternative": {
           "value": "#0260A4",
-          "description": "(blue600: #0260A4) primary.alternative should be used as an alternative to primary.default for things such as hover states",
+          "description": "(blue600: #0260A4) Reserved for \"pressed\" state on interactive elements.",
```

Again should we suggest primary elements here?
```suggestion
          "description": "(blue600: #0260A4) Reserved for \"pressed\" state on primary elements.",
```